lib/glimesh/accounts.ex

```diff
@@ -11,7 +11,19 @@ defmodule Glimesh.Accounts do
   ## Database getters
 
   def list_users do
-    Repo.all(from u in User, where: not is_nil(u.confirmed_at))
+    Repo.all(from(u in User))
```

Comment:
```suggestion
    Repo.all(from u in User, where: u.is_banned == false)
```
This'll make it skip over any globally banned users